### PR TITLE
Fix deployment scripts

### PR DESCRIPTION
**Pull request by k1rill-fedoseev**
Added the required changes to the deploy scripts, missed under #8

### CODE REVIEW

File: deploy/.env.example [hunk 1]
```diff
@@ -39,7 +39,6 @@ HOME_AMB_BRIDGE=
 FOREIGN_AMB_BRIDGE=
 
 # Request gas limit to be used when passing a message to the other side of the bridge
-HOME_MEDIATOR_REQUEST_GAS_LIMIT=
```
Comment (akolotov) [inline deploy/.env.example:42]: Does it make sense to have this limit be applied by default on the home side of the bridge?

File: deploy/README.md [hunk 2]
```diff
@@ -92,9 +92,6 @@ HOME_AMB_BRIDGE=0x
 # to the Home network.
 FOREIGN_AMB_BRIDGE=0x
 # The gas limit that will be used in the execution of the message passed to the mediator contract
-# in the Foreign network.
-HOME_MEDIATOR_REQUEST_GAS_LIMIT=2000000
```
Comment (akolotov) [inline deploy/README.md:96]: May you consider to add the forwarding rules manager deployment to be configured through the `.env` file as well? It cold be done in a separate PR.

File: deploy/.env.example (hunk 1 above)
Comment (k1rill-fedoseev) [reply deploy/.env.example]: Done. Added a module constructor argument with the default request gas limit which is taken from this env variable.

File: deploy/README.md (hunk 2 above)
Comment (k1rill-fedoseev) [reply deploy/README.md]: Sure, will do it in a separate PR.

**Review by akolotov — APPROVED** [review]
